src/main/scala/gestalt/Quote.scala

```diff
@@ -51,8 +52,10 @@ abstract class Quote(val t: Toolbox, val toolboxName: String) {
     t.Select(qual, parts.last)
   }
 
-  // lifts: m.Tree => t.Tree[t.Tree]
-
+  /** {{{
+    * AnyTree = t.Tree | t.TypeTree
+    * (trees: Seq[m.Tree[A]]) => (t.Tree[Seq[String]] | t.Tree[Seq[AnyTree[?]]])
+    * }}}*/
```

Comment:
Will change the unbound type parameters (`A`,`B`,`C`) to `_`. But still would like to leave A in for functions where it appears in both sides, because it implies a loose preservation property when converting from scalaMeta Trees to toolbox'es Trees. Would that be ok?

Will remove the unnecessary argument and parenthesis for single argument functions.

It must be able to return `t.Tree[Seq[String]]` or `t.Tree[Seq[AnyTree[_]]]`, because it is used to generate function arguments of `Seq[String]`, `Seq[t.Tree[_]]` and `Seq[t.TypeTree[_]]`
For example, `Interpolate`:
```scala
val Interpolate: InterpolateHelper
  trait InterpolateHelper {
    def apply(prefix: String, parts: Seq[String], args: Seq[Tree]): Tree
  }
selectToolbox("Interpolate").appliedTo(t.Lit(tag), liftSeq(parts), liftSeq(args))
```
And `ApplyType`:
```scala
val ApplyType: ApplyTypeHelper
  trait ApplyTypeHelper {
    def apply(fun: Tree, args: Seq[TypeTree]): Tree
    def unapply(tree: Tree): Option[(Tree, Seq[TypeTree])]
  }
selectToolbox("ApplyType").appliedTo(lift(fun), liftSeq(targs))
```

`liftSeq` __can__ return `t.Tree[Seq[String]]` only via `liftQuasi` if the there are `arg`s of type `t.Tree[String]`